src/Gang/data/tasks.ts

```diff
@@ -379,7 +379,7 @@ export const gangMemberTasksMetadata: IGangMemberTaskMetadata[] = [
     params: { chaWeight: 100, difficulty: 8 },
   },
   {
-    desc: "Members assigned to this task increase your gang's power and will fight for territory if 'Territory Warfare' is engaged. Note that gang members can be killed while assigned this task.",
+    desc: "Members assigned to this task increase your gang's power, and will fight for territory if 'Territory Battles' is engaged.<br><br>Note that gang members can be killed while assigned this task during clashes.",
```

Comment:
![image](https://github.com/bitburner-official/bitburner-src/assets/141260118/777aeeab-e653-4a0b-abcc-90ad9b6dd86c)

Reply:
Read the description aloud, and only breathe once for each comma and twice for each period. It's uncomfortable to run straight through to the end with no air. You can feel where the pauses are, and it makes it easier to parse at a glance in this case as well.

Ideally this would be broken down into more sentences with simpler structure, but we have a small space to work with here, so making it easy to glance at is probably the best we can do

Reply:
Shrug. It's a fine example of what was quoted. 2 list items, related and of equal importance joined with "and."
- Members assigned to this task increase your gang's power.
- Members will fight for territory if 'Territory Warfare' is engaged.

 "Commas where you breathe" is sort of a broad guideline, and not one that always fits proper style.

Reply:
From the AP style guide on Coordinating Conjunctions
> ...no comma when the subject of the two clauses ["Members"] is the same and is not repeated in the second: We are visiting Washington and plan to see the White House.

From TextBroker
>If your subject is only stated once, then what you have is a compound predicate. It is incorrect to split a compound predicate.

Reply:
It's not a simple case of coordinating conjunctions. The conditional phrase "if 'Territory Battles' is engaged" only applies to the 2nd clause. If there were a comma directly before the "if", it would be clear that the conditional applies to *both* clauses. With the comma as written, it's clear that it only applies to the 2nd clause. No comma would be ambiguous.

Reply:
when the _subject_ of the 2 clauses is the same, which it is. The grammatical subject in this case is "Members", stated once and shared between the clauses.

But citing 3 references is enough, if someone wants to add an error it's not worth more words.

Reply:
I'm not proposing that this fits style guides; that isn't the main goal of documentation or user interface, especially when you have limited space to convey a lot of info. I'm pointing out that it makes the sentence easier to parse and understand the first time through, which is more valuable to the user as far as I can tell.

Reply:
I mean, I noticed because it _didn't_ make it easier to parse. Improper punctuation is like that, it stands out and breaks flow.

>especially when you have limited space to convey a lot of info

A constraint we both share...
thus a stylistically appropriate way to combine two clauses into one sentence.

Better to break it in 2 than add contradictory commas. If it's difficult without a comma, it's difficult enough with one to just simplify further.

Reply:
PS - "Members assigned to this task increase your gang's power and will fight for territory if 'Territory Warfare' is engaged. Note that gang members can be killed while assigned this task."

Some online tools put this around 7th or 8th grade reading level. Sometimes material should be aimed lower than that, but in the context of a text based programming game, 8th is a pretty good target IMO.

Reply:
As someone who also enjoys language, I can't quite drop this.

> in the AP example "We are visiting Washington and plan to see the White House." Add a condition "We are visiting Washington and plan to see the White House if it's open."

This is not an identical example, because it's not ambiguous. Why? Well, because we can see that the 2nd form doesn't work: "We are visiting Washington and plan to see the White House, if it's open." Attempting to rewrite the sentence this way makes the referent of "it" more ambiguous, and the overall implication (that you will cancel your entire trip to Washington if the White House is closed) is awkward enough that it can't really be written in this form.

A better example would be "We are visiting Washington and plan to see the White House if we are well." I think this shows the *same* ambiguity, and thus the same need for a comma, as the base sentence: Either after the "and", to make it clear that the White House visit will be skipped, or before the "if", to make it clear that the entire trip will be called off.

Reply:
Found an example of text graded closer to 4-7, just for comparison
![image](https://github.com/bitburner-official/bitburner-src/assets/141260118/78790f8d-0886-4de9-8591-57b9dc303b97)